### DISCUSSION TITLE
Use it with filament RichEditor ?

### DISCUSSION THREAD

**Discussion by zumbidoweb** [awcodes/filament-curator#97] [Q&A]
how to integrate with the RichEditor uploader ?

**Comment by awcodes** [comment]
I'm not sure that's even possible with Trix's mostly limited API. I'll do some digging but I wouldn't hold my breath. If you have any docs on it I'd be happy to have a look at them.

**Comment by awcodes** [comment]
The real problem is that the functionality would have to be added to the Filament field. There's not a way to inject it into the component.

**Comment by awcodes** [comment]
It does integrate with my Tiptap editor plugin though.
